packages/flutter/lib/src/material/elevated_button.dart

```diff
@@ -164,16 +177,26 @@ class ElevatedButton extends ButtonStyleButton {
     bool? enableFeedback,
     AlignmentGeometry? alignment,
     InteractiveInkFeatureFactory? splashFactory,
+    bool useMaterial3Colors = false,
```

Comment:
I feel that having a second opt-in is confusing, and should be avoided at all costs.

Reply:
Agreed, but we can't use ThemeData.useMaterial3 here because it is a static function that doesn't have access to a BuildContext.

Reply:
Why not pass the context in instead?

Reply:
The styleFrom methods are static, so it wouldn't make sense to require a context.